### PR TITLE
SRD Update

### PR DESCRIPTION
**Pull request by jcte02**
**Changelog**
- Removed `Ageless` feat, added it as a perk (#169)
- Clarified Multi-Targeting wording (#166)
- Removed leftover HTML tags in Aura boon
- Added notes for supernatural attribute zero (#146)
- Added size categories (#163)
- Clarified bane stacking
- Addedd Area of Effect shapes
- Updated defense targeting
- Removed obsolete table regarding defense targeting
- Renamed `Supernatural` to `Extraordinary` (#161)
- Updated `Haste` boon (#158)
- Updated `Well-Rounded` feat (#155)
- Changed `10+ damage` for `10 over defense`

**TL;DR** Updates the SRD and closes most of the issues with the `SRD update` tag.

At the moment the SRD needs caring, since I found issues that were not tracked, like the addition of AoE shapes. I will take the time in the next days to read through the entire document, and follow with part 2 if needed.

### CODE REVIEW

**Comment by brianfeister** [comment]
You are my greatest hero. I've had so many things to do and this was getting close to the top of my list. Super grateful

**Comment by jcte02** [comment]
I've imagined that with the end of the Kickstarter campaign things became quite buisy. I'm trying to help out for what I can. 😄 Feel free to tag me when you need help and I'll see what I can do when I've got the time.

**Comment by ucffool** [comment]
Props. I try to stay away from the SRD changes, being new to helping and it being so important, I tend to edit other areas, so glad you've got a handle on it!

**Comment by jcte02** [comment]
**Changelog**
- Updated damage on a miss and added rationale (#159)
- Removed leftover HTML tag in Action Roll table
- Clarified relationship between XP system and GM level-up decision (#165)

Some issues that I saw and forgot to edit in the first batch.

**Comment by brianfeister** [comment]
👍
